frontend/amundsen_application/static/js/components/ResourceListItem/TableListItem/index.tsx

```diff
@@ -55,9 +55,9 @@ const TableListItem: React.FC<TableListItemProps> = ({
   tableHighlights,
   disabled,
 }) => (
-  <li className={`list-group-item ${disabled ? 'is-disabled' : 'clickable'}`}>
+  <li className="list-group-item">
     <Link
-      className="resource-list-item table-list-item"
+      className={`resource-list-item table-list-item ${disabled ? 'is-disabled' : 'clickable'}`}
```

Comment:
It seems you don't have prettier and eslint configured properly, that's why you get this error.